skimage/filters/edges.py

```diff
@@ -18,26 +18,33 @@
 
 EROSION_SELEM = generate_binary_structure(2, 2)
 
-HSOBEL_WEIGHTS = np.array([[ 1, 2, 1],
-                           [ 0, 0, 0],
-                           [-1,-2,-1]]) / 4.0
+HSOBEL_WEIGHTS = np.array([[1, 2, 1],
+                           [0, 0, 0],
+                           [-1, -2, -1]]) / 4.0
 VSOBEL_WEIGHTS = HSOBEL_WEIGHTS.T
 
-HSCHARR_WEIGHTS = np.array([[ 3,  10,  3],
-                            [ 0,   0,  0],
+HSCHARR_WEIGHTS = np.array([[3, 10, 3],
+                            [0, 0, 0],
                             [-3, -10, -3]]) / 16.0
 VSCHARR_WEIGHTS = HSCHARR_WEIGHTS.T
 
-HPREWITT_WEIGHTS = np.array([[ 1, 1, 1],
-                             [ 0, 0, 0],
-                             [-1,-1,-1]]) / 3.0
+HPREWITT_WEIGHTS = np.array([[1, 1, 1],
+                             [0, 0, 0],
+                             [-1, -1, -1]]) / 3.0
 VPREWITT_WEIGHTS = HPREWITT_WEIGHTS.T
 
 ROBERTS_PD_WEIGHTS = np.array([[1, 0],
                                [0, -1]], dtype=np.double)
 ROBERTS_ND_WEIGHTS = np.array([[0, 1],
                                [-1, 0]], dtype=np.double)
 
+p = np.array([[0.0376593171958126, 0.249153396177344, 0.426374573253687,
```

Comment:
Please add a comment here with the section of the paper where these weights can be found, since they are not straightforward :-).